src/yuzu/main.cpp

```diff
@@ -2921,12 +2904,35 @@ void GMainWindow::UpdateStatusBar() {
     emu_frametime_label->setVisible(true);
 }
 
+void GMainWindow::UpdateGPUAccuracyButton() {
+    switch (Settings::values.gpu_accuracy.GetValue()) {
+    case Settings::GPUAccuracy::Normal: {
+        gpu_accuracy_button->setText(tr("GPU NORMAL "));
+        gpu_accuracy_button->setChecked(false);
+        break;
+    }
+    case Settings::GPUAccuracy::High: {
+        gpu_accuracy_button->setText(tr("GPU HIGH   "));
+        gpu_accuracy_button->setChecked(true);
+        break;
+    }
+    case Settings::GPUAccuracy::Extreme: {
+        gpu_accuracy_button->setText(tr("GPU EXTREME"));
```

Comment:
The spacing here is rather weird, considering that the font may not be monospace
```suggestion
        gpu_accuracy_button->setText(tr("GPU NORMAL"));
        gpu_accuracy_button->setChecked(false);
        break;
    }
    case Settings::GPUAccuracy::High: {
        gpu_accuracy_button->setText(tr("GPU HIGH"));
        gpu_accuracy_button->setChecked(true);
        break;
    }
    case Settings::GPUAccuracy::Extreme: {
        gpu_accuracy_button->setText(tr("GPU EXTREME"));
```